From 1f544ff7ab906f2e77aaf2c6a32178c131774f97 Mon Sep 17 00:00:00 2001
From: James Cowgill <james410@cowgill.org.uk>
Date: Sat, 3 Dec 2016 16:32:26 +0000
Subject: [PATCH] Use LC_ALL=C to generate manpage date

Without this, the user's locale may influence the content of the
manpages (eg juin instead of June for a french locale).
---
 man/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index f889434..3cb404a 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -1,5 +1,5 @@
 .0.1:
-	@[ -z "$$SOURCE_DATE_EPOCH" ] || d=--date=@$$SOURCE_DATE_EPOCH ; sed -e "s/!VERSION!/@JACK_RELEASE@/g" -e "s/!DATE!/`date $$d '+%B %Y'`/g" < $*.0 > $@
+	@[ -z "$$SOURCE_DATE_EPOCH" ] || d=--date=@$$SOURCE_DATE_EPOCH ; sed -e "s/!VERSION!/@JACK_RELEASE@/g" -e "s/!DATE!/`LC_ALL=C date $$d '+%B %Y'`/g" < $*.0 > $@
 	@echo Built $*.1 from template
 
 manpages = $(patsubst %.0,%.1,$(wildcard *.0))
-- 
2.10.2

